src/main/java/lottery/domain/LotteryTicket.java

```diff
@@ -16,53 +17,55 @@ public LotteryTicket(List<LotteryNumber> lotteryNumbers) {
             throw new IllegalArgumentException("중복없는 6개의 숫자가 입력되어야 합니다.");
         }
 
-        this.lotteryNumbers = lotteryNumbers;
-        Collections.sort(this.lotteryNumbers);
+        this.lotteryNumbers = new TreeSet<>();
```

Comment:
```suggestion
        this.lotteryNumbers = new TreeSet<>(lotteryNumbers);
```
위와 같이 구현가능하지 않을까요  ?

Reply:
네 해당건으로 하겠습니다.
간단하게 되는걸 제가 돌아서 처리를 했네요. 👍